tsconfig.json

```diff
@@ -70,7 +66,7 @@
 
     /* Interop Constraints */
     // "isolatedModules": true,                          /* Ensure that each file can be safely transpiled without relying on other imports. */
-    // "allowSyntheticDefaultImports": true,             /* Allow 'import x from y' when a module doesn't have a default export. */
+    "allowSyntheticDefaultImports": true,                /* Allow 'import x from y' when a module doesn't have a default export. */
```

Comment:
Required for compatibility with Node ESM. In Node ESM, CommonJS `module` is always the `default` export, which is what this enforces.